site/en/workflows.md

```diff
@@ -1217,19 +1244,146 @@ copies them the a new Output fileGrp, re-generating the PAGE XML from the curren
   </tbody>
 </table>
 
+### Step X: Multi-step
+
+<!-- BEGIN-EVAL sed -n '0,/^## Notes/ p' ./repo/ocrd-website.wiki/Workflow-Guide-multistep.md|sed '$d' -->
+The procecssors in this group do more than one step in an OCR-D workflow. This
+allows these processors to optimize the data flow, e.g. reusing more precise
+internal data structures, instead of relying on the serialization of results by
+a previous step. The downside of these processors is that they are less
+interoperable with the granular processors, removing a certain amount of
+control from the user and making the implementations more complex.
+
+**Note:** All the `ocrd-tesserocr-segment*` processors internally delegate to
+`ocrd-tesserocr-recognize`, so you can replace calls to these task-specific
+processors with calls to `ocrd-tesserocr-recognize` with specific parameters:
+
+<table>
+  <thead><tr><th>processor call</th><th><code>ocrd-tesserocr-recognize</code> parameters</th></tr></thead>
+  <tbody>
+    <tr>
+      <td>ocrd-tesserocr-segment-region -P overwrite_regions true -P crop_polygons</td>
+      <td>ocrd-tesserocr-recognize -P textequiv_level none -P segmentation_level region -P overwrite_segments true -P block_polygons</td>
+    </tr>
+    <tr>
+      <td>ocrd-tesserocr-segment-table -P overwrite_cells true</td>
+      <td>ocrd-tesserocr-recognize -P textequiv_level none -P segmentation_level cell -P overwrite_segments true</td>
+    </tr>
+    <tr>
+      <td>ocrd-tesserocr-segment-line -P overwrite_lines true</td>
+      <td>ocrd-tesserocr-recognize -P textequiv_level none -P segmentation_level line -P overwrite_segments true</td>
+    </tr>
+    <tr>
+      <td>ocrd-tesserocr-segment-word -P overwrite_words true</td>
+      <td>ocrd-tesserocr-recognize -P textequiv_level none -P segmentation_level word -P overwrite_segments true</td>
+    </tr>
+  </tbody>
+</table>
+
+<!-- END-EVAL -->
+
+### Step X: Font annotation
+
+<!-- BEGIN-EVAL sed -n '0,/^## Notes/ p' ./repo/ocrd-website.wiki/Workflow-Guide-font.md|sed '$d' -->
+These processors can determine the font family (e.g. Antiqua, Fraktur,
+Schwabacher) or font style (e.g. *italic*, **bold**).
+
+**Note:** `ocrd-tesserocr-fontshape` can either use existing segmentation or segment on-demand. It can detect the following font styles:
+  * `fontSize`
+  * `fontFamily`
+  * `bold`
+  * `italic`
+  * `underlined`
+  * `monospace`
+  * `serif`
+
+**Note:** `ocrd-typegroups-classifier` can only annotate font families on page level but can detect a wider variety of fonts, including the confidence value (separated by colon). Supported `fontFamily` values:
+  * `Antiqua`
+  * `Bastarda`
+  * `Fraktur`
+  * `Gotico`-Antiqua
+  * `Greek`
+  * `Hebrew`
+  * `Italic`
+  * `Rotunda`
+  * `Schwabacher`
+  * `Textura`
+  * `other_font`
+  * `not_a_font`
+
+**Note:** `ocrd-typegroups-classifier` only works on *non-binarized* RGB images.
```

Comment:
Is that something the processor implementation can take care of (silently)?

Reply:
How do you mean "take care silently"? We could add defensive strategy to just bail out if provided bitonal images, but that would be the opposite of silent.

Reply:
At the moment, `ocrd-typegroups-classifier` does not use core's image API:

https://github.com/seuretm/ocrd_typegroups_classifier/blob/a5a205bb2f9fce533ed195c76f5f23f648acf48b/ocrd_typegroups_classifier/processor.py#L38

I have already created a [PR](https://github.com/seuretm/ocrd_typegroups_classifier/pull/16) for this long ago but received no response.

Now, when using the image API, it's of course always possible to get the raw input image, simply by adding `feature_filter="binarized"` (which is always guaranteed to yield a result).

Reply:
There has been a bit of backlog with the PR, @seuretm doesn't have time for it apparently. Since I've been making PyPI releases from unmerged PR of it, maybe we should continue development in a fork until @seuretm (hopefully) returns to maintaining it.